documentation/sphinx/source/release-notes/release-notes-7131.rst

```diff
@@ -0,0 +1,198 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.1
+======
+
```

Comment:
There are a few in the "Features" and "Reliability" section that add/improve tests. Maybe those should be under "Other" or "Testing"?

Reply:
As mentioned above, scanning past release notes `Other Changes` is a section, there isn't any `Testing` section, we can move it there if that helps

Reply:
I think it would make sense to move them to `Other Changes` in that case, but I'm also okay with leaving them as-is.